tools_for_todoist/models/event.py

```diff
@@ -100,6 +100,14 @@ def get_private_info(self, key):
             return None
         return self._extended_properties.get('private', {}).get(key)
 
+    def delete_private_info(self, key):
+        if self._extended_properties is None:
+            return
+        self._extended_properties.get('private', {}).pop(key, None)
+
+    def delete_last_completed(self):
```

Comment:
this especially might be overkill, it's not saving much and to be honest `CALENDAR_LAST_COMPLETED` probably shouldn't be part of `CalendarEvent` at all anyway, since it's specific to the syncing service.